source/style-guide/markup/format/inline.txt

```diff
@@ -0,0 +1,110 @@
+=============
+Inline Markup
+=============
+
+.. default-domain:: mongodb
+
+.. include:: /includes/styles/corrections.rst
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Basic Markup
```

Comment:
If "Basic Markup" isn't a standardized term, it might be easier to understand by using "Text Formatting Markup"